icbase/pageTSconfig.txt
RTE {
	default {
		# Define the buttons to show
		showButtons (
			formatblock, blockstyle, textstyle, class
			, bold, italic, underline
			, strikethrough, subscript, superscript
			, left, center, right, justifyfull
			, orderedlist, unorderedlist, outdent, indent
			, link, image, table
			, undo, redo, copy, cut, paste
			, spellcheck, findreplace, removeformat, showhelp
			, textindicator, insertcharacter
			, chMode
			, toggleborders, tableproperties, rowproperties, rowinsertabove, rowinsertunder, rowdelete, rowsplit, columninsertbefore, columninsertafter, columndelete, columnsplit, cellproperties, cellinsertbefore, cellinsertafter, celldelete, cellsplit, cellmerge
		)

		removeTags := removeFromList(center, font, strike, u, script)
		removeTagsAndContents := removeFromList(script)

		proc {
			# Allow embed (Youtube) HTML tags in the RTE
			allowTags := addToList(embed, iframe, object, param, script)
			allowTagsOutside := addToList(embed, iframe, object, script)

			entryHTMLparser_db {
				allowTags < RTE.default.proc.allowTags
				removeTags < RTE.default.removeTags
				removeTagsAndContents < RTE.default.removeTagsAndContents

				tags {
					# Allow style and align for paragraphs
					p {
						fixAttrib.align.unset >
						allowedAttribs := addToList(align, style)
					}
				}
			}
		}

		buttons {
			toggleborders.keepInToolbar = 1

			image {
				options {
					# realistic magic insert sizes
					magic.maxWidth = 600
					magic.maxHeight = 450
				}
			}
		}

		ignoreMainStyleOverride = 1
		showTagFreeClasses = 1

		hidePStyleItems = removeFromList(pre)

		## Configuration specific to the spellcheck button or SpellCheck feature (htmlArea RTE only)
		## Enable the use of personal dictionaries
		enablePersonalDicts = 1

		# This disables the RTE context menu so that spelling checking can be done with some browsers' context menu.
		disableContextMenu = 1
	}

	# MLC 20081030 WEC inspired settings
	default {
		buttons {
			# Disable Upload tab for media links
			# link.options.removeItems = media_upload

			image {
				options {
					# Disable Upload and Drag&Drop tabs for images
					# removeItems = upload
				}
			}
		}

		removeTagsAndContents = link, meta, title

		# clear typical styling
		classesAnchor >
		classesLinks = 
		# set classes available to these properties. Paragraph is another name
		# for block
		classesCharacter := addToList(coding, note, important, warning)
		classesParagraph := addToList(coding, note, important, warning, float_left, center, float_right)
		classesTD := addToList(sub)
		classesTable := addToList(full, fixed)

		# list all available classes here, otherwise they're removed on RTE save
		proc.allowedClasses := addToList(coding, note, important, warning, float_left, center, float_right, sub, full, fixed)

		# clear typical styling
		inlineStyle >

		# disable the “New magic image”
		blindImageOptions = magic
	}

	classes >
}

TCEFORM {
	pages {
		# MLC set 0 to 1 to turn off page item
		layout.disabled = 0
		lastUpdated.disabled = 0
		newUntil.disabled = 0

		target.disabled = 0
		no_cache.disabled = 0
		cache_timeout.disabled = 0
		
		abstract.disabled = 0
		keywords.disabled = 0
		author.disabled = 0
		email.disabled = 0
		description.disabled = 0 

		fe_login_mode.disabled = 0
		module.disabled = 0
		# image.config.disable_controls = upload
	}

	tt_news {
		# image.config.disable_controls = upload
	}

	tt_content {
		header_layout {
			altLabels.1 = Heading 1
			altLabels.2 = Heading 2
			altLabels.3 = Heading 3
			altLabels.4 = Heading 4
			altLabels.5 = Heading 5
		}
		
		# example for adding and remove header_layout items
		# header_layout.removeItems = 100
		# header_layout.addItems.6 = Heading 6
		# header_layout.addItems.99 = Hidden

		section_frame {
			# altLabels.1 = Title Bar
			# addItems.2 = Module Feature
			removeItems = 1, 5, 6, 10, 11, 12, 20, 21
		}

		# example to remove various content types
		# other options: rte, multimedia
		# CType.removeItems = div, splash, script, bullets, search

		sys_language_uid.disabled = 0
		colPos.disabled = 1
		section_frame.disabled = 0
		sectionIndex.disabled = 1
		spaceBefore.disabled = 1
		spaceAfter.disabled = 1

		header_position.disabled = 1
		date.disabled = 1
		linkToTop.disabled = 1

		bodytext.RTEfullScreenWidth = 100%
	}

	# Disable the secondary fields for the header of FCEs only
	header_position.types.templavoila_pi1.disabled = 1
	header_link.types.templavoila_pi1.disabled = 1
	header_layout.types.templavoila_pi1.disabled = 1
	date.types.templavoila_pi1.disabled = 1
}

# field names of the pages table to be displayed in the create-new-page-wizard
mod.web_txtemplavoilaM1 {
	# 1 show unlink and delete icons for local elements side-by-side
	# 2 show the delete icon and hide the unlink icon whenever possible
	enableDeleteIconForLocalElements = 1
	createPageWizard.fieldNames = hidden, nav_hide, title, tx_seo_titletag, keywords, description, abstract, author, author_email
}

mod.xMOD_alt_doc {
    disableDocSelector = 1
    disableCacheSelector = 1
}

TSFE.jumpUrl_transferSession = 1

TCEMAIN {
	translateToMessage = TRANSLATION NEEDED for %s

	# Don't hide newly copied pages or content. Prevent `copy X` on copy
	default {
		disablePrependAtCopy = 1
		disableHideAtCopy = 1
	}

	table {
		pages < .default
		tt_content < .default
	}
}


icbase/userTSconfig.txt
options {
	clearCache {
		pages = 1
		all = 1
	}

	saveDocNew = 1

	# don't clear clipboard
	saveClipboard = 1

	# Reduces the amount of clipboards from 4 to X:
	clipboardNumberPads = 2

	shortcutFrame = 1
	defaultFileUploads = 20
	
	# Enable upload field
	uploadFieldsInTopOfEB = 1

	# Enable folder create
	createFoldersInEB = 1

	## Set the default spelling ability of the check speller for all users
	## options: ultra, fast, normal, bad-spellers
	HTMLAreaPspellMode = normal

	## Enable the personal dictionary feature of the check speller by default for all users
	enablePersonalDicts = 1

	#show pageID in page tree title
	pageTree.showPageIdWithTitle = 0

	# show navigation title in page tree if any
	pageTree.showNavTitle = 1

	# options.contextMenu.pageTree.disableItems = history,edit_access,move_wizard,new_wizard,db_list,tx_impexp_clickmenu,info,perms, versioning

	options.RTEkeyList = *
	# options.RTEkeyList = formatblock, bold, italic, orderedlist, unorderedlist, outdent, indent, link, unlink, blockstylelabel, blockstyle, textstylelabel, textstyle
}

# http://typo3.org/documentation/document-library/core-documentation/doc_core_tsconfig/1.0.0/view/1/2/
setup {
	default {
		copyLevels = 9

		# hide the file upload option
		# edit_docModuleUpload = 0

		edit_RTE = 1
		edit_showFieldHelp = 0

		# Makes the BE fields wider
		# edit_wideDocument = 1

		navFrameResizable = 1
		neverHideAtCopy = 1
		noOnChangeAlertInTypeFields = 0
		recursiveDelete = 1
		startModule = web_layout
		thumbnailsByDefault = 1
		titleLen = 50
	}

	override {
		# hide the file upload option
		# edit_docModuleUpload = 0
	}
}

mod.xMOD_alt_doc {
	# If set, the document selector is disabled
    disableDocSelector = 1

	# If set, the cache/save/close selector is disabled
    disableCacheSelector = 0
}

mod.web_list {
	# edit record just by clicking on title in list-view
	clickTitleMode = edit

	# alternating colors for records in list-view
	alternateBgColors = 1

	itemsLimitPerTable = 20
	itemsLimitSingleTable = 100

	# Limit new record types
	# allowedNewTables = pages, tt_news, tt_news_cat
}

# Hide static_info_tables records on the rootpage
mod.web_list.hideTables = static_template, static_countries, static_country_zones, static_currencies, static_languages, static_territories, static_taxes, static_markets


icseo/CHANGELOG
2009-10-15 Thursday 22:36  mc

	* update CHANGELOG

2009-10-15 Thursday 22:31  mc

	* move extconf to extension; default image zoom on

2009-10-10 Saturday 11:28  mc

	* update CHANGELOG

2009-10-10 Saturday 09:09  mc

	* update CHANGELOG

2009-10-10 Saturday 09:08  mc

	* Inital import of "res" directory

2009-10-07 Wednesday 10:04  mc

	* Inital import of "README.txt" file

2009-09-25 Friday 01:20  mc

	* add headers

2009-09-24 Thursday 05:02  mc

	* dep. robots; remove sitemap; shrink page abstract

2009-09-24 Thursday 03:53  mc

	* Removal of "userTSconfig.txt"

2009-09-24 Thursday 03:53  mc

	* Removal of "pageTSconfig.txt"

2009-09-24 Thursday 03:52  mc

	* Removal of "ext_conf_template.txt"

2009-09-21 Monday 10:54  mc

	* minor cleanups

2009-09-21 Monday 09:31  mc

	* update CHANGELOG

2009-09-21 Monday 09:31  mc

	* Inital import of 'icseo' directory

2009-09-21 Monday 09:31  mc

	* Initial revision



log_analyzer/config.php
<?php
/**
 * Configuration for log_analyzer
 *
 * @author Michael Cannon <michael@peimic.com>
 * @version $Id: config.php,v 1.2 2011/05/12 10:48:30 peimic.comprock Exp $
 */
$config							= unserialize( $TYPO3_CONF_VARS['EXT']['extConf']['log_analyzer'] );

// uid of the page where FE plugin inserted
if (!defined(PAGE_UID))
   define('PAGE_UID',$config['page_uid']);

// base url for links in xls file
if (!defined(BASE_URL))
   define('BASE_URL',$config['base_url']);

// name of backend user group of whom report will be sent to
$be_group = $config['be_group'];

// Log report email settings
if (!defined(LOG_EMAIL_FROM_EMAIL))
   define('LOG_EMAIL_FROM_EMAIL',$config['log_email_from_email']);
if (!defined(LOG_EMAIL_FROM_NAME))
   define('LOG_EMAIL_FROM_NAME',$config['log_email_from_name']);
if (!defined(LOG_EMAIL_SUBJECT))
   define('LOG_EMAIL_SUBJECT',$config['log_email_subject']);
if (!defined(LOG_EMAIL_BODY))
   define('LOG_EMAIL_BODY',$config['log_email_body']);
if (!defined(IGNORE_CRON_CHANGES))
   define('IGNORE_CRON_CHANGES',$config['ignore_cron_changes']);

// default value for reports period
// currently 24 hours 
// 60 seconds * 60 minutes * 24 hours in a day
$report_period = $config['report_period'];

// EDIT NO FURTHER

// extension directory
if (!defined(EXT_ROOT))
   define(EXT_ROOT,dirname(__FILE__) .'/');

// rows in page in report
if (!defined(PAGE_ROWS))
   define('PAGE_ROWS',100);

// rows in page in report XLS
if (!defined(XLS_ROWS))
   define('XLS_ROWS',300);

// PEAR Spreadsheet directory
if (!defined(PEAR_ROOT))
   define('PEAR_ROOT', EXT_ROOT . "Spreadsheet/");

$tables_names = array(
                  'pages'       => 'Pages',
                  'tt_news'     => 'News',
                  'tt_content'  => 'Content',
                  'sys_template'  => 'TypoScript Templates',
                );

// field from which we get title
$tables_fields = array(
                  'pages'       => 'pages.title',
                  'tt_news'     => 'tt_news.title',
                  'tt_content'  => 'pages.title',
                  'be_users'    => 'be_users.username',
                );

$report_type = array(
                  'be'          => 1,
                  'view_daily'  => 2,
                  'view_custom' => 3,
                  'xls_mail'    => 4,
               );

$cron_tables = array(
                     'all',
                     //'tt_news',
                     //'tt_content'
                    );

// log enable/disable for debug uses
if (!defined(LOGGER))
   define('LOGGER',false);
?>
